consensus/safety-rules/src/error.rs

```diff
@@ -41,6 +41,8 @@ pub enum Error {
     ValidatorNotInSet(String),
     #[error("Vote proposal missing expected signature")]
     VoteProposalSignatureNotFound,
+    #[error("execution signature present, but no public key to validate it")]
```

Comment:
nits:
- Errors are ordered alphabetically in this list, can you move the error to the right location?
- Error strings should capitalize the first word (to be consistent). Could you also do that for "IncorrectRound".. not sure how it slipped through 😮